Address flaky tests of SDMX-ML/XSD validation

Some test(s) of validation appear to be flaky, failing randomly with what seem to be I/O errors.

<details>
<summary>Examples:</summary>

ubuntu-latest-py3.13 job:
```
FAILED sdmx/tests/format/test_format_xml.py::test_validate_xml_from_v2_1_samples[parts5] - lxml.etree.XMLSchemaParseError: Premature end of data in tag documentation line 334, line 334, column 236
```
</details>

To resolve:
- Investigate the root cause and fix, and/or
- Apply appropriate test marks.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -47,6 +47,7 @@ tests = [
   "pytest >= 5",
   "pytest-cov",
   "pytest-xdist",
+  "pytest-rerunfailures",
   "responses",
   "sdmx1[cache]",
 ]

--- a/sdmx/tests/format/test_xml.py
+++ b/sdmx/tests/format/test_xml.py
@@ -61,6 +61,7 @@ def test_install_schemas_invalid_version(version):
         sdmx.install_schemas(version=version)
 
 
+@pytest.mark.flaky(reruns=5)
 @pytest.mark.network
 @pytest.mark.parametrize(
     "parts",
@@ -89,6 +90,7 @@ def v30_zipball_path(installed_schemas):
     yield _extracted_zipball(Version["3.0.0"])
 
 
+@pytest.mark.flaky(reruns=5)
 @pytest.mark.network
 @pytest.mark.parametrize(
     "parts",
@@ -110,6 +112,7 @@ def test_validate_xml_from_v3_0_samples(installed_schemas, v30_zipball_path, par
     )
 
 
+@pytest.mark.flaky(reruns=5)
 @pytest.mark.network
 def test_validate_xml_invalid_doc(tmp_path, installed_schemas):
     """Ensure that an invalid document fails validation."""

